Jenkinsfile

```diff
@@ -1161,50 +1239,24 @@ pipeline {
                         label 'ci_vm9'
                     }
                     steps {
-                        unstash 'CentOS-rpm-version'
-                        script {
-                            daos_packages_version = readFile('centos7-rpm-version').trim()
-                        }
                         provisionNodes NODELIST: env.NODELIST,
                                        node_count: 9,
                                        profile: 'daos_ci',
                                        distro: 'el7',
                                        snapshot: true,
-                                       inst_repos: el7_daos_repos,
-                                       inst_rpms: 'daos-' + daos_packages_version +
-                                                  ' daos-client-' + daos_packages_version +
-                                                  ' ' + functional_rpms
-                        runTest stashes: [ 'CentOS-install', 'CentOS-build-vars' ],
-                                script: '''test_tag=$(git show -s --format=%B | sed -ne "/^Test-tag:/s/^.*: *//p")
-                                           if [ -z "$test_tag" ]; then
-                                               test_tag=pr,-hw
-                                           fi
-                                           tnodes=$(echo $NODELIST | cut -d ',' -f 1-9)
-                                           # set DAOS_TARGET_OVERSUBSCRIBE env here
-                                           export DAOS_TARGET_OVERSUBSCRIBE=0
-                                           rm -rf install/lib/daos/TESTING/ftest/avocado ./*_results.xml
-                                           mkdir -p install/lib/daos/TESTING/ftest/avocado/job-results
-                                           ./ftest.sh "$test_tag" $tnodes''',
-                                junit_files: "install/lib/daos/TESTING/ftest/avocado/*/*/*.xml install/lib/daos/TESTING/ftest/*_results.xml",
-                                failure_artifacts: 'Functional'
+                                       inst_repos: el7_daos_repos(),
+                                       inst_rpms: get_daos_packages('centos7') + ' ' +
+                                                  functional_rpms
+                        runTestFunctional stashes: [ 'CentOS-install', 'CentOS-build-vars' ],
```

Comment:
We should standardize on stash names prefixes like "centos7", etc.
For matrix builds this would be controlled by an environment variable that we can either use directly or use a function to supply the values to the step.

For non-matrix builds, we can put the name in the stage, which is in an environment variable that we can either use directly or have a function translate to what we need.

We can use functions also to create parameters passed to the dockerfile command from the stage name to make those steps exactly the same for all build steps.

Same for the parameters to provisionNodes

Reply:
> We should standardize on stash names prefixes like "centos7", etc.

As I did with the RPM version stash, where that `centos7` used there was a deliberate reuse of the artifact directory name.  These names above are historic and could be changed in a future refactor.  But that's not work for this ticket.

> For non-matrix builds, we can put the name in the stage, which is in an environment variable that we can either use directly or have a function translate to what we need.
> 
> We can use functions also to create parameters passed to the dockerfile command from the stage name to make those steps exactly the same for all build steps.
> 
> Same for the parameters to provisionNodes

Sure.  But again, all work for a refactoring effort, not this effort to be able to test with RPMs.